raiden/utils/nursery.py

```diff
@@ -146,6 +148,16 @@ def _free_resources(self) -> None:
             for p in self._processes:
                 p.send_signal(signal.SIGINT)
 
-            for p in self._processes:
-                if p.wait() != STATUS_CODE_FOR_SUCCESS:
-                    print("Process did not exit cleanly", p.communicate())
+            try:
+                for p in self._processes:
+                    exit_code = p.wait(timeout=20)
+                    if exit_code != STATUS_CODE_FOR_SUCCESS:
+                        log.warning(
+                            "Process did not exit cleanly",
```

Comment:
Would be nice to log some information about the process that failed to shut down properly.

Reply:
I've added `p.args` to the output. For the SP this looks like
```
2020-03-09 14:32:46.402045 [warning  ] Process did not stop in time. Sending SIGKILL to all remaining processes! [raiden.utils.nursery] command=['/home/karl/scenario-player/venv/bin/raiden', '--accept-disclaimer', '--datadir', '/home/karl/.raiden/scenario-player/scenarios/pfs1_get_a_simple_path/node_125_003', '--keystore-path', '/home/karl/.raiden/scenario-player/scenarios/pfs1_get_a_simple_path/node_125_003/keys', '--address', '0x9e015dfA8CDe054A19B0169D5BA1Bf98eBF14A4e', '--password-file', '/home/karl/.raiden/scenario-player/scenarios/pfs1_get_a_simple_path/node_125_003/password.txt', '--network-id', '5', '--sync-check', '--gas-price', 'fast', '--eth-rpc-endpoint', 'http://parity.goerli.ethnodes.brainbot.com:8545', '--log-config', ':info,raiden:debug,raiden_contracts:debug,raiden.api.rest.pywsgi:warning', '--log-json', '--log-file', '/home/karl/.raiden/scenario-player/scenarios/pfs1_get_a_simple_path/node_125_003/run-125.log', '--disable-debug-logfile', '--matrix-server', 'auto', '--api-address', '127.0.0.1:45541', '--no-web-ui', '--pathfinding-service-address', 'https://pfs-goerli-with-fee.services-dev.raiden.network', '--gas-price', 'fast', '--routing-mode', 'pfs', '--proportional-fee', '0x59105441977ecD9d805A4f5b060E34676F50F806', '0', '--environment-type', 'development', '--pathfinding-max-paths', '5', '--pathfinding-max-fee', '100', '--proportional-imbalance-fee', '0x59105441977ecD9d805A4f5b060E34676F50F806', '0']

```